javascript-obfuscator/javascript-obfuscator.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for javascript-obfuscator
+// Project: https://github.com/sanex3339/javascript-obfuscator
+// Definitions by: sanex3339 <https://github.com/sanex3339>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare module 'javascript-obfuscator' {
+  export class JavaScriptObfuscator {
+    public static obfuscate (sourceCode: string, customOptions?: any): string;
```

Comment:
It's be cool if you included an options definition for the 'customOptions' parameter and maybe renamed the parameter 'options' like it is over on the [project's readme](https://github.com/sanex3339/javascript-obfuscator#available-options)
i.e. something like (based on the project's readme):
```ts
options?: {
  compact?: boolean;
  debugProtection?: boolean;
  debugProtectionInterval?: boolean;
  disableConsoleOutput?: boolean;
  rotateUnicodeArray?: boolean;
  wrapUnicodeArrayCalls?: boolean;
  [id: string]: any;
}
```